server-go/project_storage.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 The Grafeas Authors. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package server
+
+import (
+	pb "github.com/grafeas/grafeas/v1alpha1/proto"
+)
+
+// ProjectStorager is the interface that a Grafeas storage implementation would provide
+// for Project enteties
+type ProjectStorager interface {
+	// CreateProject adds the specified project
+	CreateProject(pID string) error
```

Comment:
these interface methods should have a context as the first parameter. With context, you go all in. Annoying to plumb through, but the idea is that perhaps the underlying .CreateProject() call also makes an RPC to write the data - it would want to time out on the parent context as well.

Reply:
Sounds like a good idea but I don't think it is within scope of this PR as this would require corresponding changes to `Storager` as well. Would also like @R2wenD2  to approve of this before changing the interface. Will open a new issue for this instead.